pkg/builder/builder.go

```diff
@@ -184,8 +187,12 @@ func buildVolumeMounts(vdb *vapi.VerticaDB) []corev1.VolumeMount {
 		volMnts = append(volMnts, buildSSHVolumeMounts()...)
 	}
 
-	if vdb.Spec.NmaTLSSecret != "" {
-		volMnts = append(volMnts, buildHTTPServerVolumeMount()...)
+	if vmeta.UseVClusterOps(vdb.Annotations) {
+		if vmeta.UseNMACertsMount(vdb.Annotations) {
```

Comment:
Could we combine the lines?
```suggestion
	if vmeta.UseVClusterOps(vdb.Annotations) && vmeta.UseNMACertsMount(vdb.Annotations) {
```